### PR TITLE
docs(clouddriver/sql): Add release notes for fix to clouddriver sql cache. 

### PR DESCRIPTION
**Pull request by mattsanta**
docs(clouddriver/sql): Add release notes for fix to clouddriver sql cache. 

Release notes for Clouddriver SQL Cache data too long issue fix (https://github.com/spinnaker/spinnaker/issues/5600). Since we've moved to a new version of the tables used for caching we need to instruct users how to delete the old version once they are ready.

### CODE REVIEW

**Comment by mattsanta** [comment]
PR for fix that should make it in before release branch cut: https://github.com/spinnaker/clouddriver/pull/4755, but let's wait until it's been merged in order to merge this doc PR.

**Comment by mattsanta** [comment]
@maggieneterval who recommended adding this change to the release notes :)

File: community/releases/next-release-preview/index.md
Comment (maggieneterval) [inline community/releases/next-release-preview/index.md:74]: did we determine that only admins can hit these endpoints? might be worth highlighting!

**Review by maggieneterval — APPROVED** [review]

File: community/releases/next-release-preview/index.md
Comment (dorbin) [inline community/releases/next-release-preview/index.md]:
```suggestion
### Issue Resolved: Clouddriver SQL Cache Data Too Long
```

File: community/releases/next-release-preview/index.md
Comment (dorbin) [inline community/releases/next-release-preview/index.md]:
```suggestion
We've fixed an issue that prevented data from being stored in Clouddriver's SQL 
```

File: community/releases/next-release-preview/index.md
Comment (dorbin) [inline community/releases/next-release-preview/index.md]:
```suggestion
Once you're comfortable with the Spinnaker release and don't expect to roll back 
```

File: community/releases/next-release-preview/index.md
Comment (dorbin) [inline community/releases/next-release-preview/index.md]:
```suggestion
to a previous version, then you can delete the first version of the 
```

File: community/releases/next-release-preview/index.md
Comment (dorbin) [inline community/releases/next-release-preview/index.md]:
```suggestion
endpoint can be reached via a `curl PUT` request against 
```

**Review by dorbin — APPROVED** [review]
Some edits, but LGTM. Thanks Matt!

File: community/releases/next-release-preview/index.md
Comment (mattsanta) [reply community/releases/next-release-preview/index.md:74]: Yep, when validating the permissions it explicitly checks if the user is an admin.

File: community/releases/next-release-preview/index.md
Comment (mattsanta) [reply community/releases/next-release-preview/index.md]: Done!

File: community/releases/next-release-preview/index.md
Comment (mattsanta) [reply community/releases/next-release-preview/index.md]: Done!

File: community/releases/next-release-preview/index.md
Comment (mattsanta) [reply community/releases/next-release-preview/index.md]: Done!

File: community/releases/next-release-preview/index.md
Comment (mattsanta) [reply community/releases/next-release-preview/index.md]: done!

File: community/releases/next-release-preview/index.md
Comment (mattsanta) [reply community/releases/next-release-preview/index.md]: done, thanks for all the comments!

**Pull request merged**
